docs/core/introduction.md

```diff
@@ -8,167 +8,65 @@ recommendations: false
-
-## Tools and productivity
-
-.NET gives you a choice of languages, integrated development environments (IDEs), and other tools.
+* [Cloud native apps](../architecture/cloud-native/index.md)
+* [Console apps](tutorials/with-visual-studio-code.md)
+* [Serverless functions in the cloud](/azure/azure-functions/functions-create-first-function-vs-code?pivots=programming-language-csharp)
+* [Web apps, web APIs, and microservices](/aspnet/core/introduction-to-aspnet-core#recommended-learning-path)
 
-### Programming languages
+**Cross-platform client apps**
 
-.NET supports three programming languages:
+* [Desktop apps](https://dotnet.microsoft.com/apps/desktop)
+* [Games](https://dotnet.microsoft.com/apps/games)
+* [Mobile apps](https://dotnet.microsoft.com/apps/maui)
 
-* [C#](../csharp/index.yml)
+**Windows apps**
 
-  C# (pronounced "See Sharp") is a modern, object-oriented, and type-safe programming language. C# has its roots in the C family of languages and will be immediately familiar to C, C++, Java, and JavaScript programmers.
+* [Windows Desktop apps](https://dotnet.microsoft.com/apps/desktop)
+  * [Windows Forms](/dotnet/desktop/winforms/)
+  * [Windows WPF](/dotnet/desktop/wpf/)
+  * [Universal Windows Platform (UWP)](/windows/uwp/get-started/create-a-hello-world-app-xaml-universal)
+* [Windows services](/aspnet/core/host-and-deploy/windows-service)
 
-* [F#](../fsharp/index.yml)
+**Other app types**
 
-  F# is an open-source, cross-platform, interoperable programming language for writing succinct, robust and performant code. Your focus remains on your problem domain, rather than the details of programming. F# programming is data-oriented, where code involves transforming data with functions.
-  
-* [Visual Basic](../visual-basic/index.yml)
+* [Machine learning](../machine-learning/index.yml)
+* [Internet of Things (IoT)](../iot/index.yml)
 
-  Among the .NET languages, the syntax of Visual Basic is the closest to ordinary human language, which can make it easier to learn. Unlike C# and F#, for which Microsoft is actively developing new features, the Visual Basic language is stable. Visual Basic isn't supported for web apps, but it is supported for web APIs.
+## Features
 
-Here are some of the capabilities that .NET languages support:
+.NET includes many features that enable developers to productively write reliable and performant code.
 
-* [Type safety](../standard/base-types/common-type-system.md)
-* Type inference - [C#](../csharp/fundamentals/types/index.md#specifying-types-in-variable-declarations), [F#](../fsharp/language-reference/type-inference.md), [Visual Basic](../visual-basic/programming-guide/language-features/variables/local-type-inference.md)
-* [Generic types](../standard/generics.md)
-* [Delegates](../standard/delegates-lambdas.md)
-* [Lambdas](../standard/delegates-lambdas.md)
+* [Asynchronous code](../csharp/programming-guide/concepts/async/index.md)
+* [Attributes](../standard/attributes/index.md)
```

Comment:
This list should include Reflection

Reply:
```suggestion
* [Attributes](../standard/attributes/index.md)
* Reflection
```

It appears that we only have a .NET Framework version of this doc. Logged an issue https://github.com/dotnet/docs/issues/29741